tests/scripts/auto-grow-storage.sh

```diff
@@ -179,7 +179,7 @@ function growOSD(){
 function creatingPrerequisites(){
    echo "creating Prerequisites deployments - Prometheus Operator and Prometheus Instances"
    # creating Prometheus operator
-   kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.40.0/bundle.yaml
+   kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.71.1/bundle.yaml
```

Comment:
```suggestion
   kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.71.1/bundle.yaml
```

One suggestion, please use global variable for the version and use that version in all the code places so that we'll have central place to maintain. Thanks

Reply:
It might be simpler for future updates if we have the same URL to search and replace in the docs and scripts. It sounds strange not to recommend a variable for the version, but in this case it seems appropriate.